source/members.rst

```diff
@@ -4,11 +4,12 @@
 PyPA Members, And How To Join
 =============================
 
-:Last Reviewed: 2020-06-08
+:Last Reviewed: 2020-07-20
 
-Historically, the PyPA has been a loose confederation of people
+Historically, the PyPA was a loose confederation of people
 and projects; the word "authority" in the name was meant to be
-a joke.
+a joke. :pep:`609` formalizes the PyPA somewhat and clarifies
```

Comment:
Maybe let's drop a word?

```suggestion
a joke. :pep:`609` formalizes the PyPA and clarifies
```

Reply:
No, I think it's still only partly formal and I'm fine with this phrasing. :-) Thanks for the suggestion, though.